django/conf/global_settings.py

```diff
@@ -524,6 +524,7 @@
 CSRF_COOKIE_PATH = '/'
 CSRF_COOKIE_SECURE = False
 CSRF_COOKIE_HTTPONLY = False
+CSRF_USE_SESSIONS = False
```

Comment:
If session-CSRF will require a different approach for AJAX, then that documentation needs to be updated (to show both techniques) regardless of what the startproject template contains. The only different might be a matter of emphasis (which approach is documented first and more prominently). I don't think we should remove the documentation for the existing cookie-based approach.